package.json

```diff
@@ -6,6 +6,8 @@
   "devDependencies": {
     "@babel/core": "^7.13.10",
     "@babel/preset-env": "^7.13.12",
+    "@testing-library/dom": "^7.30.3",
+    "@testing-library/user-event": "^13.1.2",
```

Comment:
Using the `userEvent.keyboard` API in the test for this change. `@testing-library/dom` is a peer dependency of `@testing-library/user-event`.